cli/src/main/resources/help/reload.txt

```diff
@@ -2,7 +2,7 @@ SYNOPSIS
 
    Standalone mode:
 
-      reload [--admin-only=true|false] [--use-current-server-config=true|false]
+      reload [--admin-only=true|false] [--use-current-server-config=true|false] [--server-config=new_server_config_file_name]
```

Comment:
I think this needs a line break. These are meant to fit within a certain number of columns per line.